funnel/views/profile.py

```diff
@@ -242,6 +243,61 @@ def view(self) -> ReturnRenderWith:
 
         return ctx
 
+    def jsonify_calendar(jsonData):
+        return jsonify(jsonData['projects'])
```

Comment:
Several problems:

1. As a class method, the first parameter should be `self`, or the method should be decorated with `@staticmethod`
2. The parameter name should be in Python convention, snake-cased: `json_data`
3. This method can be entirely replaced with a lambda function since it has a single line